.github/workflows/capture-gallery-manual.yml

```
@@ -41,8 +41,8 @@ jobs:
           path: |
             docs/images/**
 
-      - name: Optional: Commit images to main (requires PAT)
-        if: ${{ secrets.GALLERY_PUSH_TOKEN != '' }}
+      - name: "Optional: Commit images to main (requires PAT)"
+        if: '${{ secrets.GALLERY_PUSH_TOKEN }}'
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)